buildout.cfg

```diff
@@ -68,6 +68,7 @@ twine = 1.8.1
 z3c.recipe.mkdir = 0.6
 zc.recipe.egg = 2.0.3
 zest.releaser = 6.6.4
+django-import-export = 1.2.0
```

Comment:
That one looks to be pretty well-maintained and active.

I looked at it because it is a dependency that will be installed in all our SSO-using websites (so: almost all of them).

Reply:
Yeap, really nice package for administrative stuffs :)